### ISSUE TITLE
Block backends need more flexible

### ISSUE THREAD

**Issue by luqmana** [oxidecomputer/propolis#491]

**Comment by leftwo** [comment]
> Crucible:  It may also sometimes initiate Flush without a corresponding guest/device request but need to double check.

Yes, crucible will issue a flush a regular intervals if it does not get one from the guest.
